packages/frontend/src/common/components/ArticleBox.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode } from "react";
+import { Box } from "@mui/material";
+
+interface ArticleBoxProps {
+  children: ReactNode;
+}
+export default function ArticleBox({ children }: ArticleBoxProps) {
+  return (
+    <Box sx={{ justifyContent: "center", display: "flex", marginTop: "2rem" }}>
+      <Box maxWidth="lg">{children}</Box>
+    </Box>
```

Comment:
Can't we merge these two boxes?

Reply:
We need both boxes. The outer box aligns the content to be centered, where as the inner box wraps the actual content